app/components/build/AvatarWithUnknownEmailPrompt.js

```diff
@@ -46,7 +46,9 @@ class AvatarWithUnknownEmailPrompt extends React.Component {
     hasBeenDismissed: false
   };
 
-  getUserEmail(email) {
+  // Given an email address, try and return the associated UserEmail GraphQL
+  // node
+  findUserEmailNode(email) {
```

Comment:
`getUserEmail` reminded me more of a "getter" type method - so I switched it to `find` to kind of hint that you may get nil, and to be more indicative of what it's doing.